### PR TITLE
[SMALLFIX] Added test coverage for lineage store and id generator

### PR DESCRIPTION
**Pull request by yupeng9**

### CODE REVIEW

File: common/src/main/java/tachyon/Constants.java [hunk 1]
```diff
@@ -305,7 +305,8 @@
   public static final String LOGGER_TYPE = System.getProperty(TACHYON_LOGGER_TYPE, "");
   public static final String ACCESS_LOGGER_TYPE =
       System.getProperty(TACHYON_ACCESS_LOGGER_TYPE, "");
-  public static final boolean DEBUG = Boolean.valueOf(System.getProperty(TACHYON_DEBUG, "false"));
+  public static final boolean DEBUG =
+      Boolean.parseBoolean(System.getProperty(TACHYON_DEBUG, "false"));
```
Comment (calvinjia) [inline common/src/main/java/tachyon/Constants.java]: Why does this show up in the diff?

**Comment by yupeng9** [comment]
Merged with master. @calvinjia , PTAL

**Comment by calvinjia** [comment]
Jenkins, test this please.

File: servers/src/test/java/tachyon/master/lineage/meta/LineageStoreTest.java [hunk 2]
```diff
@@ -61,4 +72,66 @@ public void completeFileForAsyncWriteTest() {
...
+  }
+
+  @Test
+  public void commitNonexistingFilePersistenceTest() {
+    long fileId = 1;
+    mThrown.expect(IllegalStateException.class);
+    mThrown.expectMessage(String.format(PreconditionMessage.LINEAGE_NO_OUTPUT_FILE, fileId));
+
+    mLineageStore.commitFilePersistence(fileId);
+  }
```
Comment (calvinjia) [inline servers/src/test/java/tachyon/master/lineage/meta/LineageStoreTest.java]: Should we also add the tests for the scenarios that work?

**Comment by calvinjia** [comment]
Jenkins, test this please.

**Comment by yupeng9** [comment]
@calvinjia , added, PTAL

**Comment by yupeng9** [comment]
Jenkins, test this please

File: common/src/main/java/tachyon/exception/PreconditionMessage.java [hunk 3]
```diff
@@ -42,4 +42,7 @@
   public static final String ERR_WRITE_BUFFER_NULL = "Cannot write a null input buffer";
   public static final String REMOTE_CLIENT_BUT_LOCAL_HOSTNAME =
       "Acquire Remote Worker Client cannot not be called with local hostname";
+  public static final String LINEAGE_NO_OUTPUT_FILE =
+      "The output file %s is not associated with any lineage";
+  public static final String LINEAGE_NOT_EXIST = "Lineage id %s does not exist";
```
Comment (jsimsa) [inline common/src/main/java/tachyon/exception/PreconditionMessage.java]: nit: `LINEAGE_NOT_EXISTS`

File: servers/src/test/java/tachyon/master/lineage/meta/LineageIdGeneratorTest.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+package tachyon.master.lineage.meta;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import tachyon.master.journal.JournalEntry;
+import tachyon.master.lineage.journal.LineageIdGeneratorEntry;
+
+/**
+ * Tests lineage id generator
```
Comment (jsimsa) [inline servers/src/test/java/tachyon/master/lineage/meta/LineageIdGeneratorTest.java]: nit: add the missing `.`

**Comment by jsimsa** [comment]
Just minor nits. Overall it looks good to me.

**Comment by calvinjia** [comment]
LGTM

File: common/src/main/java/tachyon/exception/PreconditionMessage.java (hunk 3 above)
Comment (calvinjia) [inline common/src/main/java/tachyon/exception/PreconditionMessage.java]: Could we spare the extra word and go with `LINEAGE_DOES_NOT_EXIST`?

Comment (yupeng9) [inline common/src/main/java/tachyon/exception/PreconditionMessage.java]: done

File: servers/src/test/java/tachyon/master/lineage/meta/LineageStoreTest.java (hunk 2 above)
Comment (yupeng9) [inline servers/src/test/java/tachyon/master/lineage/meta/LineageStoreTest.java]: done

**Comment by gpang** [comment]
Overall looks good, but could you resolve conflicts? Thanks!

**Comment by yupeng9** [comment]
@gpang, resolved. PTAL.

**Comment by gpang** [comment]
LGTM

**Pull request merged**